### PR TITLE
feat: Oauth docs

### PR DESCRIPTION
**Pull request by vsukhin**
## Pull request description 

OAuth docs

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: docs/oauth.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Testkube can automatically create OAuth2-Proxy service and deployment integrated 
+with GitHub, as well as properly configure Kubernetes Nginx Ingress Controller and create required 
+ingresses.
+
+# Provide Parameters for UI and API Ingresses
+
+## API Ingress
+
+Pass values to Testkube Helm chart during installation or upgrade (they are empty by default).
+Pay attention to the usage of the scheme (http or https) in uris.
```
Comment (jfermi) [inline docs/oauth.md:14]: Should this be "scheme" or "schema"?

File: docs/oauth.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+--set testkube-api.ingress.enabled=true \
+--set testkube-api.ingress.annotations."nginx\.ingress\.kubernetes\.io/auth-url"="http://\$host/oauth2/auth" \
+--set testkube-api.ingress.annotations."nginx\.ingress\.kubernetes\.io/auth-signin"="http://\$host/oauth2/start?rd=\$escaped_request_uri" \
+--set testkube-api.ingress.annotations."nginx\.ingress\.kubernetes\.io/access-control-allow-origin" = "*"
+```
+
+## UI Ingress
+
+Pass values to Testkube Helm chart during installation or upgrade (they are empty by default).
+Pay attention to the usage of the scheme (http or https) in uris.
```
Comment (jfermi) [inline docs/oauth.md:26]: Should this be "scheme" or "schema"?

File: docs/oauth.md (hunk 1 above)
Comment (vsukhin) [reply docs/oauth.md:14]: I beleive, it's scheme https://en.wikipedia.org/wiki/List_of_URI_schemes

File: docs/oauth.md (hunk 2 above)
Comment (vsukhin) [reply docs/oauth.md:26]: Same as above

File: docs/oauth.md (hunk 1 above)
Comment (jfermi) [reply docs/oauth.md:14]: Got it.

File: docs/oauth.md (hunk 2 above)
Comment (jfermi) [reply docs/oauth.md:26]: Got it.

**Review by jfermi — APPROVED** [review]

**Pull request merged**